src/scss/components/ec-card/_ec-card.scss

```diff
@@ -4,7 +4,35 @@
 
   padding: 8px;
   background: $white;
-  overflow: hidden;
+  word-break: break-word;
+
+  &--is-interactive {
+    position: relative;
+    cursor: pointer;
+    transition: all 0.4s ease;
```

Comment:
We should not use `transition: all` because that's not very future proof. It will cause colors, paddings, margins to transition too, which is not a case right now but it might change in the future.

Just specify the property you have intention to do the transition for, just like you had it in the initial commit.

Reply:
Good point @mcibique I'll do it, thanks!